### PR TITLE
Feature pipelines

### PR DESCRIPTION
**Pull request by YannickBrombach**
Removing the namespaces deletion during the cleanup phase

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -371,31 +371,31 @@ stages:
...
-      displayName: Delete AKS namespace and Kusto database
-      condition: and(variables['KUBERNETES_NAMESPACE'], not(eq(variables['KUBERNETES_NAMESPACE'], 'master')))
-      inputs:
-        azureSubscription: $(ACR_PULL_SERVICE_CONNECTION)
-        scriptLocation: inlineScript
-        inlineScript: |
-          set -eux  # fail on error
-          kubectl delete namespace "$KUBERNETES_NAMESPACE"
-          az kusto database delete -g "$(RESOURCE_GROUP)" --cluster-name "$(KUSTO_NAME)" -n "$(KUSTO_DB)" -y
+# - stage: cleanup_integration_tests
```
Comment (eladiw) [inline azure-pipelines.yml:374]: commenting this code, will cause the integration tests to never be deleted.
We need to add an exclusion condition (e.g. line 391)

**Review by eladiw — CHANGES_REQUESTED** [review]
Added comment

File: azure-pipelines.yml (hunk 1 above)
Comment (YannickBrombach) [reply azure-pipelines.yml]: Fixed by adding a condition instead

**Review by eladiw — CHANGES_REQUESTED** [review]
Search for the work 'master' in this file. I think some (but not all) of the places you find needs to be addressed with vnext (example: line 252)

**Review by eladiw — APPROVED** [review]
lgtm. lets try it

**Review by AsafMah — APPROVED** [review]

**Pull request merged**